### PR TITLE
Cans 1030 test fix

### PR DESCRIPTION
**Pull request by DmitriyRudenko**
## Tests

- [ ] I used TDD to develop the feature
- [ ] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I ran all my tests locally which were green.
- [ ] My code adds no new issues to Code Climate
- [ ] I ran all the linters for the project.
- [ ] My code is in a stable state ready to be deployable, but not necessarily complete.
- [ ] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by DmitriyRudenko** [comment]
retest this please

File: spec/page_objects/assessment_form.rb [hunk 1]
```diff
@@ -230,6 +228,15 @@ def fill_conducted_by_last_name_field(text)
   )
 end
 
+def set_value_for_text_field(field, text)
+  field.click
+  sleep 2
```
Comment (wmitchell-cambria) [inline spec/page_objects/assessment_form.rb]: Is this just a temporary fix, or a proposed final solution?

**Comment by wmitchell-cambria** [comment]
It looks like the fields are causing a large amount of re-rendering on every keystroke. This would definitely explain why Capybara has issues entering in values quickly. We should fix the rendering issues themselves instead of adding sleeps to the test files.

Render Highlighting

File: spec/page_objects/assessment_form.rb (hunk 1 above)
Comment (DmitriyRudenko) [reply spec/page_objects/assessment_form.rb]: temp fix

Comment (DmitriyRudenko) [reply spec/page_objects/assessment_form.rb]: definitely something wrong with capybara...

**Comment by DmitriyRudenko** [comment]
@wmitchell-cambria thanks for addressing this, I'll take a look, I think main reason is onAssessmentUpdate callback

**Pull request merged**